democracy_club/templates/hermes/post_detail.html

```diff
@@ -11,6 +11,12 @@
 {% block og_title_content %}{{ post.subject }}{% endblock og_title_content %}
 {% block og_type_content %}article{% endblock og_type_content %}
 
+ <!-- Twitter -->
+{% block og_image_twitter_content %}{% thumbnail post.hero "690" as im %}{{ im.url }}{% endthumbnail %}{% endblock og_image_twitter_content %} 
+{% block og_twitter_description_content %}{{ post.summary }}{% endblock og_twitter_description_content %}"/>
+
+<script src="{% static 'js/global.js' %}"></script>
```

Comment:
I don't think this is needed / used any more?

It's included in `base.html`, so I think can be removed from here.